### PR TITLE
Add support for Python 3.7.

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,13 +49,14 @@ def read(*rnames):
...
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
         'Natural Language :: English',
         'Operating System :: OS Independent',
         'Topic :: Internet :: WWW/HTTP',
-        'Framework :: Zope3'],
-    url='http://pypi.python.org/pypi/z3c.authviewlet',
+        'Framework :: Zope :: 3'],
```
Comment (mgedmin) [inline setup.py:58]: We have "Framework :: Zope3" in 209 zopefoundation packages.

We have "Framework :: Zope :: 3" in 3 packages (z3c.checkversions, zodbupdate and zope.fanstatic).

I think we should stick with the more popular classifier.

**Review by mgedmin — APPROVED** [review]
LGTM, except for the framework classifier change.

(If we want to deprecate one classifier and switch to another, I think it would be best to do a mass commit to all 209 affected repos.  I'd be happy to volunteer for that part, given justification for the change.  The annoying bit would be making 209 releases and I have no plans for volunteering for _that_ part. ;-)

File: setup.py (hunk 1 above)
Comment (icemac) [reply setup.py:58]: The more popular classifier is the older one which I am going to declare as deprecated. (To get a Trove classifier for Zope 4 we had to switch to the new schema.) My plan is to migrate all usages of the old classifier to the new format. My idea was to do this every time I see an old classifier.

I do not have checkouts for all 209 repositories so I would appreciate it very much if you could
help with a mass commit to make the migration faster. I do not think that we will need releases right now. For me it is enough that the packages will be released eventually when it is needed for a feature or bugfix.

Comment (mgedmin) [reply setup.py:58]: Fair enough!

I wrote [ghcloneall](https://pypi.org/project/ghcloneall/) so I could get checkouts of all 380 zopefoundation's repos.  Given that it's easy to run a shell loop that changes setup.py files with sed and git pushes.

Comment (mgedmin) [reply setup.py:58]: I would like some meta-repository where we could track cross-repo issues like this.  https://github.com/zopefoundation/zopefoundation/issues maybe?

**Comment by icemac** [comment]
@mgedmin Thanks a lot. It is good to see that we have to fix many projects to support current releases of zope.schema, zope.configuration resp. persistent.

File: setup.py (hunk 1 above)
Comment (icemac) [reply setup.py:58]: Good idea. I decided for `meta` as project name, so there is now zopefoundation/meta#1.

**Pull request merged**